packages/adaptive_scaffold/README.md

```diff
@@ -1,18 +1,35 @@
 <?code-excerpt path-base="excerpts/packages/adaptive_scaffold_example"?>
 
-# Helper Widgets for Making Adaptive Layouts in Flutter (AdaptiveScaffold)
+# Helper Widgets for Making Adaptive Layouts in Flutter (`AdaptiveScaffold`)
 
-This package contains some helper widgets that make the process of developing adaptive layouts easier, especially with navigational elements.
+This package contains some helper widgets that make the process of developing
```

Comment:
Let's be concise about the pain points here:

> `AdaptiveScaffold` reacts to input from users, devices and screen elements and renders your Flutter application according to the [Material 3](https://m3.material.io/foundations/adaptive-design/overview) guidelines. 
> 
> /* insert demo video here*/
>

Reply:
We might need a technical writer on this...

Reply:
I reworded it, but I think I'll leave the video for another time...